chainer/training/extensions/graph_report.py

```diff
@@ -0,0 +1,141 @@
+from chainer import reporter
+import chainer.serializer as serializer_module
+from chainer.training import extension
+import chainer.training.trigger as trigger_module
+import json
+import numpy as np
```

Comment:
Could you make it `import numpy` and replace all `np` with `numpy` in this source file?